plugin/context.go

```diff
@@ -47,7 +47,7 @@ type GdmPluginContext struct {
 	DryRun     bool   `drone:"env=PLUGIN_DRY_RUN"`
 
 	// gcloud:
-	Token   string `drone:"env=PLUGIN_TOKEN"`
+	Token   string `drone:"env=TOKEN"`
```

Comment:
I believe this will be passed as `PLUGIN_TOKEN`, have you confirmed via tests?

*In either case*:
I also notice in [drone-gke/main.go](https://github.com/NYTimes/drone-gke/blob/master/main.go) they are pulling the token from the default credentials/service account - we should do this (probably best not to expose the token by env var now that plugin invocation has changed - I think their `TOKEN` parameter is parsed for _testing purposes_, but is not the way the credentials are passed during invocation).

Reply:
GKE is using the same name `TOKEN` instead of `PLUGIN_TOKEN ` https://github.com/NYTimes/drone-gke/blob/b66f189cedceb8a091a4bb986161f5a9fcb3c4c1/main.go#L85-L89

Reply:
Will merge and cut a new release this AM. Thanks @Vartlok.